### ISSUE TITLE
Clash between two libs

### ISSUE THREAD

**Issue by guillaumeLepape** [jupyter-xeus/cpp-terminal#122]
Hello,

I'm using your library for a small project. It works well for what I want to do. However, I have an error because I'm using an other library for reading, writing and modifying json (https://github.com/nlohmann/json). This lib declares two variables UTF8_ACCEPT and UTF8_REJECT that are also declared in your library. Since the json library variables are declared as static constexpr and yours using #define, compilation fail. I manage to fix this issue by declaring the two variables in your lib as static constexpr as follow, at lines 448 and 449 of `terminal.h`:  
```cpp
static constexpr std::uint8_t UTF8_ACCEPT = 0;
static constexpr std::uint8_t UTF8_REJECT = 0xf;
```

Since the json library is quite common lib in projects, this minor change can be done to avoid clash between the libs.

Regards.
Guillaume Le Pape

**Comment by certik** [comment]
@guillaumeLepape thanks a lot for your bug report!

Can you please submit a PR with this change? I agree we should not use `#define` precisely for the reason you found out.

**Comment by guillaumeLepape** [comment]
Thanks, pr is done.

**Comment by certik** [comment]
Thank you! Fixed in #123.
